service/src/test/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsServiceTest.java

```diff
@@ -351,7 +351,7 @@ void shouldReturnEmptyMapWhenNoHearingsAvailable() {
 
         @Test
         void shouldPullCustomAddressFromHearingWhenHearingVenueIsOther() {
-            given(hearingVenueLookUpService.getHearingVenue(any())).willCallRealMethod();
+            given(hearingVenueLookUpService.getHearingVenue((HearingBooking) any())).willCallRealMethod();
```

Comment:
why need casting?

Reply:
There are two functions for getHearingVenue, I made one of them public so I can look up the venue by supplying a String rather than a HearingBooking. I had to cast this so the test knew which function to use

Reply:
Corrected